templates/download/risc-v/tab-7.html

```diff
@@ -0,0 +1,43 @@
+<div tabindex="0"
+     role="tabpanel"
+     id="visionfive-2-board-tab"
+     aria-labelledby="starfive-visionfive-board"
+     hidden="true">
+    <div class="row u-vertically-center u-no-padding">
+        <div class="col-4">
+            <h2>赛昉VisionFive 2</h2>
+            <ul class="p-list--divided">
+                <li class="p-list__item">
+                    <h3 class="p-heading--4">Ubuntu Server预安装镜像</h3>
+                    <p>
+                        <a class="p-button--positive"
+                           href="https://cdimage.ubuntu.com/releases/{{ releases.lts.full_version }}/release/ubuntu-{{ releases.lts.full_version }}-preinstalled-server-riscv64+visionfive.img.xz">下载 {{ releases.lts.full_version }}</a>
+                    </p>
+                </li>
+                <li class="p-list__item">
+                    <h3 class="p-heading--4">Ubuntu Server在线安装程序</h3>
```

Comment:
Indentation looks a bit off on this file. You may want to talk to Akbar maybe to get that sorted to do it automatically?